aep/general/0135/aep.md.j2

```diff
@@ -71,24 +93,40 @@ message DeleteBookRequest {
 ```
 
 - A `name` field **must** be included. It **should** be called `name`.
-  - The field **should** be [annotated as required][aip-203].
-  - The field **must** identify the [resource type][aip-123] that it
+  - The field **should** be [annotated as required][aep-203].
+  - The field **must** identify the [resource type][aep-123] that it
     references.
 - The comment for the field **should** document the resource pattern.
 - The request message **must not** contain any other required fields, and
   **should not** contain other optional fields except those described in this
-  or another AIP.
+  or another AEP.
+
+{% endtabs %}
 
 ### Soft delete
 
 **Note:** This material was moved into its own document to provide a more
-comprehensive treatment: AIP-164.
+comprehensive treatment: [AEP-164][aep-164].
 
 ### Long-running delete
 
-Some resources take longer to delete a resource than is reasonable for a
+Some resources take longer to delete than is reasonable for a
 regular API request. In this situation, the API **should** use a long-running
-operation instead:
+operation instead. See [AEP-151][aep-151] for full details on long-running operations.
+
+{% tab oas -%}
+
+{% sample 'long_running_delete.oas.yaml', 'paths' %}
+
+- The response status code should be `202 Accepted` if the request was
+  accepted for later processing. When the request is processed it could still fail.   
+- The `response` field of the response body **must** be set to the appropriate return type if the RPC
+  was not long-running: `google.protobuf.Empty` for most Delete RPCs, or the
```

Comment:
You probably didn't mean to leave a reference to `google.protobuf.Empty` here...what's the OAS equivalent?